6.1/6.1/Program.cs

```diff
@@ -0,0 +1,51 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace _6._1
+{
+    internal class Program
+    {
+        static void Main(string[] args)
+        {
+            Console.WriteLine("Введите текст на английском:");
+            string TextEnglish = Console.ReadLine();
+            Console.WriteLine("Вот перевод вашего текста на Leet: " + LeetToTranslate(TextEnglish));
+        }
+
+        static string LeetToTranslate(string SomeTxt)
+        {
+            SomeTxt = SomeTxt.ToUpper();
+            return SomeTxt.Replace("A", "4")
+                .Replace("B", "8")
+                .Replace("C", "(")
+                .Replace("D", "|)")
+                .Replace("E", "3")
+                .Replace("F", "|=")
+                .Replace("G", "6")
+                .Replace("H", "|-|")
+                .Replace("I", "!")
+                .Replace("G", ")")
+                .Replace("K", "|<")
+                .Replace("L", "1")
+                .Replace("M", "^^")
+                .Replace("N", "ท")
+                .Replace("O", "0")
+                .Replace("P", "|>")
+                .Replace("Q", "9")
+                .Replace("R", "|2")
+                .Replace("S", "5")
+                .Replace("T", "7")
+                .Replace("U", "|_|")
+                .Replace("V", "v")
+                .Replace("W", "vv")
```

Comment:
См. комментарий выше
```suggestion
                .Replace("V", "\\/")
                .Replace("W", @"\/\/")
```